.claude/skills/wf-01-brainstorming/spec-document-reviewer-prompt.md

```
@@ -0,0 +1,48 @@
+# 仕様ドキュメントレビュアーのプロンプトテンプレート
+
+spec-document-reviewer サブエージェントをディスパッチする際にこのテンプレートを使用してください。
+
+**目的:** 仕様が完全で、一貫性があり、実装計画の作成に十分な状態か検証する。
+
+**ディスパッチタイミング:** 仕様ドキュメントが doc/specs/ に書き込まれた後
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)